src/components/Header/header.js

```diff
@@ -16,30 +24,53 @@ class Header extends Component  {
         };
 
         this.responsiveUtils = new ResponsiveUtils();
+        
     }
     componentDidMount(){
-
+        
         // On Resize and On Orientation Change Handler added to ResponsiveUtils Object
         this.responsiveUtils.__proto__.onResize = () => {
             this.responsiveUtils.viewPortSelector();
             this.setState ({
-                viewport : this.responsiveUtils.viewPort
+                viewport : this.responsiveUtils.viewport
             })
         }
         // Component starts listening to 'resize' and 'onOrientationChange' events
         this.responsiveUtils.init();
 
-        // Get the value of viewport on component mount
+        // Set the value of viewport on component mount
         this.responsiveUtils.onResize();
+
     }
 
     render(){
+
+        this.imageOptimizer = new ImageOptimize({
```

Comment:
Look online for another way to do it instead of placing this part in render function of the component.

Reply:
Fixed